singer-io/tap-adroll
# Tdl 15660 implement dev mode

# Description of change
Added support for -dev / --dev CLI param in tap-adroll,

1. Stores and uses the current refresh_token and access_token  in config while running in dev-mode
2. Prevent refresh_token and access_token expiry while running in dev-mode
3. Added unit tests to validate dev mode implementation
4. Added unit tests step in config.yaml

# Manual QA steps

-  First, run the tap normally using the standard params
- Check the config file for added keys access_token & refresh_token
- Run the tap using the --dev or '-dev' param
- Check the log file for dev mode-related logs

 
# Risks
 - Tap will exit for existing connections when run in dev mode if access_token is not available/expired (access token expiry 24 Hrs)
 
# Rollback steps
 - revert this branch
